test/interface.jl

```diff
@@ -1,12 +1,6 @@
 using SymbolicUtils, Test
 using TermInterface
 
-TermInterface.istree(ex::Expr) = ex.head == :call
-TermInterface.operation(ex::Expr) = ex.args[1]
-TermInterface.arguments(ex::Expr) = ex.args[2:end]
-TermInterface.similarterm(x::Type{Expr}, head, args, symtype=nothing; metadata=nothing) = 
-    Expr(:call, head, args...)
-
```

Comment:
How does this work now?

Reply:
https://github.com/JuliaSymbolics/TermInterface.jl/blob/master/src/expr.jl